src/uproot/interpretation/identify.py

```diff
@@ -110,9 +110,8 @@ def _from_leaves_one(leaf, title):
     m = _title_has_dims.match(title)
     if m is not None:
         dims = tuple(int(x) for x in re.findall(_item_dim_pattern, title))
-        if dims == ():
-            if leaf.member("fLen") > 1:
-                dims = (leaf.member("fLen"),)
+        if dims == () and leaf.member("fLen") > 1:
+            dims = (leaf.member("fLen"),)
```

Comment:
Sometimes, nested `if` instead of a single predicate with `and` is for code symmetry. I don't see that in this case, though.